plutus-core/plutus-core/src/PlutusCore/Value.hs

```diff
@@ -5,6 +5,10 @@
 
 module PlutusCore.Value (
   Value, -- Do not expose data constructor
+  K, -- Do not expose data constructor
+  k,
```

Comment:
Will users need to use this in Plinth?  If so, `k` might be a bit short: it could clash with a variable name, for instance.